src/core/migrations/0012_savedview_agent_updated_at.py

```
@@ -0,0 +1,73 @@
+        ),
+        migrations.AlterField(
+            model_name='agent',
+            name='updated_at',
+            field=models.DateTimeField(default=django.utils.timezone.now, editable=False),
+        ),
+        migrations.AlterField(
+            model_name='media',
+            name='updated_at',
+            field=models.DateTimeField(default=django.utils.timezone.now, editable=False),
+        ),
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)